pandas/tests/groupby/test_groupby.py

```diff
@@ -1690,14 +1690,11 @@ def test_groupby_agg_ohlc_non_first():
         [1, 1, 1, 1, 1],
         [1, 1, 1, 1, 1]
     ], columns=pd.MultiIndex.from_tuples((
-        ('foo', 'ohlc', 'open'), ('foo', 'ohlc', 'high'),
-        ('foo', 'ohlc', 'low'), ('foo', 'ohlc', 'close'),
-        ('foo', 'sum', 'foo'))), index=pd.date_range(
+        ('foo', 'sum', 'foo'), ('foo', 'ohlc', 'open'),
```

Comment:
So the only change in behavior is that the output order is now (consistently) the arguments passed to `.agg`?

Since we did `['sum', 'ohlc']` the output order is `'sum', 'open', 'high', 'low', 'close'`?

If so, let's add an entry to the release notes, under the Groupby bug fix section, saying that `.agg` now respects the order of arguments when `'ohlc'` is one of the aggfuncs.

Reply:
This might be rather difficult to express in a whatsnew since it's really just applicable to `_aggregate_multiple_funcs` here. 

Perhaps a follow up to consistently use OrderedDict's in the groupby module is in order?

Reply:
Is `_aggregate_multiple_funcs` called by just by functions like `ohlc` that have multiple outputs per input? Or is it also called by `.agg([funcs])`?

Reply:
The latter. So this has implications to ordering which previously would have been non-deterministic in Py35. 

So I think it might make sense as a follow up to use a OrderedDict and make better guarantees about the returned column order, but don't think that should hold this one up

Reply:
See #25692 as follow up